redwood-layout-view/src/main/kotlin/app/cash/redwood/layout/view/ViewFlexContainer.kt

```diff
@@ -37,16 +39,31 @@ internal class ViewFlexContainer(context: Context, direction: FlexDirection) {
     flexDirection = direction
   }
 
-  private val _view = HostView(context)
-  val view: View get() = _view
+  private val hostView = HostView(context)
+
+  private val scrollView = if (direction.isHorizontal) {
```

Comment:
Is direction fixed? I thought it was a property that could change over time.

Reply:
Nope! It's only set on initialization of the column/row.

Reply:
Oh right I guess you would have to use a relocatable content lambda and then switch between `Row` and `Column` composables.